packages/malloy-interfaces/thrift/malloy.thrift

```diff
@@ -453,11 +408,13 @@ struct NumberCell {
 struct NullCell {}
 
 struct TimestampCell {
-  1: required string timestamp_value, // TODO another way to represent dates?
+  // TODO another way to represent dates?
+  1: required string timestamp_value,
 }
 
 struct DateCell {
-  1: required string date_value, // TODO another way to represent dates?
+  // TODO another way to represent dates?
+  1: required string date_value,
 }
 
 struct JSONCell {
```

Comment:
Re: https://github.com/malloydata/malloy/pull/2327/files#diff-9d0111becc433e2a89f480d46840047e845f6816c95d1b0ecd573ca173e5827eR441

Do `TimestampCell`s need to be different from `DateCell`s?

Reply:
Mtoy: consider whether it is awkward to use if they're separate? maybe a utility function?